src/components/molecules/DeleteButtonGroup.vue

```diff
@@ -1,13 +1,31 @@
 <template>
   <div class="DeleteButtonGroup">
-    DeleteButtonGroup
+    <ApplyButton>Delete</ApplyButton>
```

Comment:
```suggestion
    <ApplyButton class="DeleteButtonGroup_Button">Delete</ApplyButton>
```

このようにしたほうが柔軟で、命名規則にも沿っているのでいいと思う